### PR TITLE
[TACHYON-440] Dump base virtualbox vm with necessary software installed

### PR DESCRIPTION
**Pull request by uronce-cc**
please see the README.md.

### CODE REVIEW

**Comment by calvinjia** [comment]
Thanks for the patch! Does it make sense to eventually share this base box?

**Comment by uronce-cc** [comment]
It's possible to share it, but currently, the process of building and installing the base box is invisible to users of deploy module, just run `run_vb.sh`, the first time, the box will be built, afterwards, it will be used.

File: deploy/vagrant/vbox/Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  config.vm.box = "chef/centos-6.5"
+  config.vm.provision :shell, path: "provision.sh"
```
Comment (calvinjia) [inline deploy/vagrant/vbox/Vagrantfile]: Is `provision.sh` provided?

**Comment by calvinjia** [comment]
+1 LGTM

**Pull request merged**